infrastructure/makefile

```diff
@@ -0,0 +1,15 @@
+
+
+login:
+	
+	@aws sso login --profile aaq
+
+data-db-tunnel-demo:
+	# Open a connection to the remote db via the bastion host
+	# the target is the instance id of the bastion host
+	@aws ssm start-session \
+	--target i-0ec0d70ab12bb6c97 \
```

Comment:
Should we make this xxxxxx as before for now?

Reply:
Im thinking of not pushing this file if the repo will be public. We can have a temp file with dummy data of how if should look like

Reply:
Sounds good!